pkg/Microsoft.Private.PackageBaseline/project.json

```diff
@@ -0,0 +1,246 @@
+{
+  "frameworks": {
+    "netstandard1.6": {
```

Comment:
For now it's checked in.  Wes and I were thinking about ways to generate it during a restore, along with generating other project.jsons :smile:  not handling that now.